assets/sass/07-utilities/ie.scss

```diff
@@ -1,45 +0,0 @@
-.is-IE {
```

Comment:
Why is this removed? I thought this was for any dark background, not specifically for the device dark mode

Reply:
You're right. added back :+1: